src/styles/SearchResults.css

```diff
@@ -0,0 +1,45 @@
+.card-img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 20%;
+}
+
+.search-results__no-results {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50vh;
+  text-align: center;
+  color: #c70019;
+  font-size: 2rem;
+  padding: 2rem;
+}
+.search-results {
+  margin-top: 2rem;
+  display: flex;
+}
+.search-results__wrapper {
+  padding: 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 4rem;
+}
+.search-results__card-image {
+  object-fit: cover;
+  height: 400px;
+  width: 400px;
+  border-radius: 5px;
+  box-shadow: 0 15px 30px -10px rgba(0, 0, 0, 0.25);
+}
+@media (max-width: 1100px) {
```

Comment:
Media queries are better placed inside the class they are effecting, in this case inside the `.search-results__card-image` class. This makes it easier to see how media queries relate to specific classes and makes them harder to lose track of, especially in larger files